### PR TITLE
TASK: Install UFW

### PR DESCRIPTION
**Pull request by Corvan**

### CODE REVIEW

**Review by fnkr — CHANGES_REQUESTED** [review]
The `Update apt cache` task must run before this (otherwise it will break on newly created servers). We should change the execution order in `tasks/main.yaml` and run `apt.yaml` before `ufw.yaml`.

**Comment by fnkr** [comment]
@Corvan You might want to add your GPG key to GitHub so it doesn't show as "Unverified".

**Comment by Corvan** [comment]
I have my gpg key in gitlab and github and both show me as unverified although I did everything in the documentation, I don't know where my error is there

File: tasks/ufw.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- name: Install ufw
```
Comment (fnkr) [inline tasks/ufw.yaml]:
```suggestion
- name: Install UFW
```

**Pull request merged**